eZ/Publish/Core/settings/tests/common.yml

```diff
@@ -1,3 +1,33 @@
+parameters:
+    ezplatform.ezrichtext.custom_tags:
+        video:
+            template: 'MyBundle:FieldType/RichText/tag:video.html.twig'
```

Comment:
Does Core know about bundles? normally we place this kind of config in the bundles don't we?

Reply:
> Does Core know about bundles? normally we place this kind of config in the bundles don't we?

This is just config for tests (`tests/common.yml`), so no need to specify anything meaningful. Actual config could be in meta-repository or in bundle and should use semantic config nodes (e.g. by prepending extension config in bundle). 
Here I put a little bit of workaround setting parameter directly to avoid complicating config and loading compiler pass.

Moreover at this point it doesn't matter much what is placed here, because this gets passed directly to Twig and Twig finds proper loaders (so if you have ezdesign and prepend it with `@ezdesign` you'll get proper path).

Reply:
> This is just config for tests (tests/common.yml), 

my bad, did not see the `tests/` folder, you can safely ignore this then ;)